change/@uifabric-experiments-2019-08-12-10-13-48-v-mare-ComboBox-fluent-updates.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Combobox: Updated web fluent styles",
+  "type": "patch",
```

Comment:
The only thing that changed in experiments package is a snapshot update. Not totally sure but I think because we don't ship the snapshots we should have type `none` here. @jdhuntington any thoughts? Plus the comment is not reflecting the true change.

Reply:
Thanks @Vitalius1 ! Switched over to type none and changed comment to read "Updated ComboBox snapshots"